packages/axios-extension/src/factory.ts

```diff
@@ -73,7 +74,7 @@ function createInstance<T extends ServiceProvider>(
         rejectUnauthorized: !providerConfig.ignoreCertErrors
     };
     const localProxy = getProxyForUrl(config.baseURL);
```

Comment:
Can we add a comment here please explaining why we are using this "localProxy"  and using it breaks the BAS proxy host resolution?

Reply:
BAS is no longer in play, remove the `proxy` property from the provider works.